packages/collapsible/Collapsible.tsx

```diff
@@ -0,0 +1,99 @@
+import { h, Component, prop, emit } from 'skatejs';
+import styles from './Collapsible.scss';
+import { css } from '../../utils/css';
+
+// public
+interface CollapsibleProps extends JSX.HTMLProps<HTMLElement | any> {
+  isOpened?: boolean,
+  isLast?: boolean,
+  onStateChanged?: (this: this, ev: StateChangedEvent) => any,
+}
+
+interface StateChangedEventDetail {
+  readonly opened: boolean;
+  readonly collapsed: boolean;
+}
+export interface StateChangedEvent extends Event {
+  readonly detail: StateChangedEventDetail
+}
+
+export class Collapsible extends Component<CollapsibleProps> {
+  static get is(){ return 'bl-collapsible' }
+  static get props() {
+    return {
+      isOpened: prop.boolean({
+        attribute: true
+      }),
+      isNotStandAlone: prop.boolean(),
+      isLast: prop.boolean()
+    }
+  }
+
+  isOpened: boolean;
+
+  // internal property to be used from other components, when there is collection of collapsible items to indicate
+  // this is the last one in the collection
+  isLast: boolean;
```

Comment:
As it is not really private, but like library internal, I would not suggest using `_`.